src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt

```diff
@@ -135,7 +135,7 @@ class ChristmasEventPlugin : JavaPlugin() {
             }
             it.billboard = Display.Billboard.CENTER
             it.isDefaultBackground = false
-            it.backgroundColor = Color.fromRGB(94, 68, 23)
+            it.backgroundColor = Color.fromARGB(0x1A000000)
```

Comment:
Again, same comment with any other occurrence of `0x`.

Also, would you be able to find a new, pastel red colour for this text, and a similar pastel for the background.

Reply:
> Also, would you be able to find a new, pastel red colour for this text, and a similar pastel for the background.

Is a pastel-like color palette also what you are aiming for? Because then I can make changes accordingly to that.

And about the actual red color, here is a palette of possible pastel-like red colors: https://artincontext.org/colorpalette/index.html?colors=DA2C38-F2545B-FF5555-FF7477-F08080-FF8B94-F4989C. **Please do not that these colors look very different when not put side-by-side like this.**